packages/wrangler/src/api/mixedMode/index.ts

```diff
@@ -24,10 +26,7 @@ export async function startMixedModeSession(
 		config: proxyServerWorkerWranglerConfig,
 		dev: {
 			remote: true,
-			auth: {
-				accountId: await requireAuth({}),
-				apiToken: requireApiToken(),
-			},
+			auth: options?.auth,
```

Comment:
why? wouldn't it make sense for `startMixedModeSession` to also handle the authentication? 🤔

Reply:
This is just allowing the user to override the authentication. If they don't pass anything in, the `ConfigController` will provide a default value (which is the same as the one we had specified here)

Reply:
ah ok, nice! thanks 🙂